shoes-core/lib/shoes/widget.rb

```diff
@@ -21,40 +21,41 @@ class Shoes
   #     say_hello 'Shoes'
   #   end
   #
-  class Widget
-    include Common::Inspect
-
+  class Widget < Shoes::Flow
```

Comment:
why do we inherit from flow and not Slot? Cause flow is the default, makes sense, just checking though

Reply:
I checked the behavior on the old Widgets, and they acted like a Flow.